GLMakie/src/postprocessing.jl

```diff
@@ -285,8 +286,7 @@ function to_screen_postprocessor(framebuffer, shader_cache, screen_fb_id = nothi
     pass.postrenderfunction = () -> draw_fullscreen(pass.vertexarray.id)
 
     full_render = screen -> begin
-        fb = screen.framebuffer
-        w, h = size(fb)
```

Comment:
There is also a window hint

> `GLFW_SCALE_TO_MONITOR` specified whether the window content area should be resized based on the [monitor content scale](https://www.glfw.org/docs/3.3/monitor_guide.html#monitor_scale) of any monitor it is placed on. This includes the initial placement when the window is created. Possible values are GLFW_TRUE and GLFW_FALSE.

which we do have available. You could try adding it here
https://github.com/MakieOrg/Makie.jl/blob/2fcee32cad5938b50046289f1b3edba3d3b80325/GLMakie/src/screen.jl#L200-L213

If this changes the size of the final framebuffer you will probably need to adjust this to something like `2 .* size(screen)`. (I'm guessing `GLFW.GetFramebufferSize(screen.glscreen)` will return the correct size, but it would be better to avoid calling that all the time. If we don't need `px_per_unit` we can probably just revert to the old code.)

Reply:
I don't have a second monitor to test with, but it sounds reasonable that pixel density should be automatic and depend on the current monitor. Unless you want to override it, or render out hi res pngs.